protocols/Love_Lab/SDS-PAGE-Prep_Biedermann_2018-07-09.ot2.py

```diff
@@ -0,0 +1,152 @@
+from opentrons import instruments, labware

```

Comment:
Heya! Double-check on the 'flattening` and transposing funcs.

Reply:
Function work for OT2